buildSrc/src/main/kotlin/com/toasttab/protokt/shared/ProtobufBuild.kt

```diff
@@ -65,3 +66,10 @@ internal fun configureProtobufPlugin(project: Project, ext: ProtoktExtension, bi
         }
     }
 }
+
+private fun normalizePath(binaryPath: String) =
+    if (OperatingSystem.current().isWindows) {
+        binaryPath.replace('\\', '/') + ".bat"
```

Comment:
I'd prefer we just built the path with File.separator here: https://github.com/open-toast/protokt/blob/master/buildSrc/src/main/kotlin/LocalProtoktBuild.kt#L23.

Reply:
yes, and then you still have to replace \ with / because protoc wants /  :rage3:

Reply:
Can we leave a comment to that effect?